### ISSUE TITLE
Normal vectors for Player representation.

### ISSUE THREAD

**Issue by PiotrJustyna** [PiotrJustyna/zero#9]
Feed the shader with hand-calculated normals to make experimenting with diffuse light easy.

**Comment by PiotrJustyna** [comment]
Done with 85bce26fce21c70510ceed916360814ec243f2b7.

The shader is now fed normal-representing lines (2 vertices each) and is able to display normal vectors (as those lines). This is a proof of concept which should eventually by extended with just normal vectors (not lines!), represented a single vertices (one model vertex - one normal vertex).

There are a couple of reasons I decided not to feed the shader with real normal vectors (single vertex per normal) yet:
- I don't really need it yet and when I do - I have my proof of concept ready to work with and expand.
- Visual representations of normal vectors (the two-vertex lines I'm using now) will be a great aid when I start replacing hand-calculated normal vectors with automatically calculated ones.

One more thing is that for now I'm representing normal vectors as e.g.:

`V4 0.0 0.0 0.1 1.0`

This of course should be:

`V4 0.0 0.0 1.0 1.0`

but since I'm not using a scaling transformation matrix at the moment and model and its normals share the same transformations, I decided that it would be just easier if I manually scale normals. This will probably change once I have an idea for scaling-per-vertex-set.

To sum up, my shader now accepts two inputs:
- model vertices
- vertices representing normal vectors (those are not normal vectors, though!)

![model and normals](https://raw.githubusercontent.com/PiotrJustyna/zero/master/images/15.png)
